inc/admin/network/class-networksettings.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * @author  Pressbooks <code@pressbooks.com>
+ * @license GPLv3 (or any later version)
+ */
+// @phpcs:disable Pressbooks.Security.EscapeOutput.OutputNotEscaped
+// @phpcs:disable Pressbooks.Security.NonceVerification.Missing
+// @phpcs:disable Pressbooks.Security.ValidatedSanitizedInput.MissingUnslash
+
+namespace Pressbooks\Admin\Network;
+
+class NetworkSettings {
+
+	const DEFAULT_THEME_OPTION = 'pressbooks_default_book_theme';
+
+	const DEFAULT_THEME = 'pressbooks-malala';
+
+	/**
+	 * @var array
+	 */
+	private $customOptions = [];
+
+	/**
+	 * @var NetworkSettings
+	 */
+	private static $instance = null;
+
+	public function __construct() {
+		$this->customOptions = [
+			'default_theme' => self::DEFAULT_THEME_OPTION,
+		];
+	}
+
+	/**
+	 * @return NetworkSettings
+	 */
+	public static function init() {
+		if ( is_null( self::$instance ) ) {
+			self::$instance = new self();
+			self::hooks( self::$instance );
+		}
+		return self::$instance;
+	}
+
+	/**
+	 * @param NetworkSettings $obj
+	 */
+	public static function hooks( NetworkSettings $obj ) {
+		add_filter( 'wpmu_options', [ $obj, 'renderCustomOptions' ] );
+		add_action( 'update_wpmu_options', [ $obj, 'saveNetworkSettings' ] );
+	}
+
+	/**
+	 * Render custom network settings options
+	 *
+	 * @return void
+	 */
+	public function renderCustomOptions() : void {
```

Comment:
For more options to render and save we would iterate over the `$this->customOptions` array, instead pick the `'default_theme'` option one.